microcosm_flask/runserver.py

```diff
@@ -15,6 +15,7 @@ def parse_args(graph):
     parser.add_argument("--host", default="127.0.0.1")
     parser.add_argument("--port", type=int, default=default_port)
     parser.add_argument("--with-profiling", action="store_true", default=default_enable_profiling)
+    parser.add_argument("--debug", action="store_true", default=False)
```

Comment:
I started with using `graph.metadata` to toggle flags in the microcosm graph object itself, but it didn't get there. I can remove this, but setting `metadata.debug` might be useful for other features.....